clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/converters/UpsertGoogleScalingPolicyAtomicOperationConverter.groovy

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
This class, and related classes, should be renamed to reflect `GoogleScaling` -> `GoogleAutoscaling`.